README.md

```diff
@@ -49,6 +49,15 @@ $uri = 'https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim'
 (New-Object Net.WebClient).DownloadFile($uri, $ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath("~\vimfiles\autoload\plug.vim"))
 ```
 
+### Getting Help
+
+- See the [requirements] page for debugging information & tested configurations.
+- See the [FAQ] for common problems and questions.
+- Make an issue.
```

Comment:
A minor nitpick. ["Create an issue"](https://help.github.com/articles/creating-an-issue/) sounds more familiar to me.